web/frontend/legacy/lib/ui/content/export_modal.js

```diff
@@ -51,15 +53,17 @@ function downloadFile (includeAnnotations, experimental=false, aws_lambda=false,
       + (includeAnnotations === "true" ? '?annotations=true' : '?annotations=false')
       + (experimental ? '&experimental=true' : '')
       + (aws_lambda ? '&aws_lambda=true' : '')
-      + (docx_footnotes ? '&aws_lambda=false&docx_footnotes=true' : ''));
+      + (docx_footnotes ? '&aws_lambda=false&docx_footnotes=true' : '')
```

Comment:
The `&aws_lambda=false` should be taken out here, I think the code has been updated to remove the non-lambda pathway.

Reply:
Right— will do.

Reply:
good to go